getinsomnia/insomnia
# bump: @sentry/electron

Bump sentry sdk version so that we can use some new features such as `custom metric`, `App no response`, etc.
Follow this migration doc: https://github.com/getsentry/sentry-javascript/blob/develop/MIGRATION.md

**Changes**

- [x] use transportOptions.shouldSend to do offline support
- [x] use Sentry.getCurrentScope().setUser({id}) to set user info 

**How to test**
change `sampleRate` to 1 in our sentry config, and trigger Sentry.captureException. Then you can see the record in sentry platform.
